.github/actions/restore-node/check-git-status.sh

```diff
@@ -1,11 +1,26 @@
 #!/bin/bash
+
 # Set verbose execution
 set -x
+
 # Navigate to the specified directory
 cd "$1" || exit $?
+
+# Get the value of IGNORE_ENDO_BRANCH
+ignore_endo_branch=$2 || exit $?
```

Comment:
_low priority_

Worth setting a default value for this variable?

Reply:
It has a default value of `false` which is set [over here](https://github.com/Agoric/agoric-sdk/blob/master/.github/actions/restore-node/action.yml#L16-L19) in the file.